kuma/users/jobs.py

```diff
@@ -0,0 +1,29 @@
+import hashlib
+import urllib
+
+from django.conf import settings
+
+from kuma.core.jobs import KumaJob
+
+
+DEFAULT_AVATAR = getattr(settings, 'DEFAULT_AVATAR',
+                         settings.MEDIA_URL + 'img/avatar-default.png')
```

Comment:
Do both paths exist? I would try to just have one default. That's too confusing.